rpc/ethereum/backend/backend.go

```diff
@@ -810,10 +811,10 @@ func (e *EVMBackend) SendTransaction(args evmtypes.TransactionArgs) (common.Hash
 	// NOTE: If error is encountered on the node, the broadcast will not return an error
 	syncCtx := e.clientCtx.WithBroadcastMode(flags.BroadcastSync)
 	rsp, err := syncCtx.BroadcastTx(txBytes)
-	if err != nil || rsp.Code != 0 {
-		if err == nil {
-			err = errors.New(rsp.RawLog)
-		}
+	if rsp != nil && rsp.Code != 0 {
```

Comment:
&& is correct?

Reply:
yes, it try to recover error from `rsp` if `rsp.Code != 0`.